### ISSUE TITLE
Accessibility: Format 3rd Party Gradebook header tags

### ISSUE THREAD

**Issue by ktowneUM** [tl-its-umich-edu/canvas-course-manager-next#651]
Based on the [CCM Accessibility Review](https://docs.google.com/spreadsheets/d/1mYrbqwFLLXQoGRTY_XEGkfS3cu3BO_pUN_zQ_DFdhUY/edit?gid=1709548063#gid=1709548063):

Severity: Major

Feature: Format 3rd Party Gradebook

Issue notes:
File: filename.csv" should be p not h5

Criterion:
2.4.10 Section Headings
